nuxt-i18n.config.ts

```diff
@@ -106,4 +105,4 @@ const options: NuxtConfig = {
   seo: false,
 }
 
-export default options as NuxtVueI18n.Options.AllOptionsInterface
+export default options as NuxtAppOptions
```

Comment:
```suggestion
export default options
```
`as XXX` にすると型チェックが利かなくなる場合があるので、32行目あたりの代入時に型を指定します

```diff
- const options: NuxtConfig = {
+ const options: NuxtConfig['i18n'] = {
```